metricbeat/module/elasticsearch/_meta/README.md

```diff
@@ -0,0 +1,62 @@
+# Elasticsearch Module
+
+- [About the elasticsearch metricbeat module](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-elasticsearch.html)
+- [List of exported fields](https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-elasticsearch.html) stored by this module generated from fields.yml files
+
+## Metricsets
+
+Metricbeat will call the following Elasticsearch API endpoints corresponding to each metricset.  The module stores a subset of the response in fields which can be found [here](https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-elasticsearch.html).  The Metricbeat exported fields generally follw the same paths as the elasticsearch API response, but may differ somewhat according to the [schema in each metricset](https://github.com/elastic/beats/blob/main/metricbeat/module/elasticsearch/node/data.go#L36).  They are [namespaced with `{module}.{metricset}`](https://github.com/elastic/beats/blob/main/metricbeat/module/elasticsearch/cluster_stats/data.go#L39).
+### ccr
+- `/_ccr/stats`
```

Comment:
I think this is a fine way to go for now, but I'd really like to see if we can get the specific paths into a unified code location like you can see in the kibana module https://github.com/elastic/beats/blob/main/metricbeat/module/kibana/kibana.go#L37-L39

That way we won't have to worry as much about keeping markdown and golang in sync over time.

Again, not something I'd block this PR on, just something I'd like us to keep in mind while we build out the module docs over time.